playbooks/roles/gitea/tasks/main.yaml

```diff
@@ -61,6 +61,14 @@
     mode: 0755
     checksum: "{{ gitea_checksum }}"
 
+- name: Write /etc/hosts entry for ldap
+  become: true
+  ansible.builtin.lineinfile:
+    path: "/etc/hosts"
+    regexp: "{{ gitea_ldap_hostname }}$"
+    line: "{{ gitea_ldap_ip }} {{ gitea_ldap_hostname }}"
+  when: gitea_add_ldap_hosts_entry
```

Comment:
I would change the condition to check presence of the attributes themselves rather then some additional flag. Otherwise the chance is big we have gitea_add_ldap_hosts_entry set to true but i.e. gitea_ldap_hostname is unset

Reply:
done